.prettierrc.js

```diff
@@ -7,4 +7,10 @@ module.exports = {
   overrides: [
     {files: '**/*.{scss,css,pcss,html,mdx}', options: {printWidth: 120}},
   ],
+  importOrderParserPlugins: [
+    'typescript',
+    'jsx',
+    JSON.stringify(['decorators', {decoratorsBeforeExport: true}]),
+  ],
+  importOrder: ['^[./]', '^\\/', '^\\.\\.\\/', '^\\.\\/'],
```

Comment:
This is my personal preference. External modules first, and then local last.

I believe it aligns most closely with what we already had.